### ISSUE TITLE
Tile Palette doesn't show up after switching to another Camera View State

### ISSUE THREAD

**Issue by ChristianGreiner** [AdamsLair/duality#499]
### Summary

Sometimes the window of the tile palette doesn't show up anymore after you switched to another camera view state.

The window also doesn't show up when I try to open it via the "view" menu strip again.

The editor doesn't throw any errors or warnings.

In my case, I tabbed the tile palette window to the scene view window

![](http://i.imgur.com/HKgsaT5.png)

**Comment by ilexp** [comment]
Can't reproduce this so far. The tile palette will be closed when I exit the Tilemap Editor (which is intended in cases when it was opened automatically before), but it comes back when I enter the Tilemap Editor again.

Can you provide a more detailed step-by-step guide to reproduce this?

**Comment by ChristianGreiner** [comment]
@ilexp So I did the following:

My editor window management looks like this:
![](http://i.imgur.com/B4M1puA.png)

1. Step:
I dragged & dropped the tile palette into the first "Scene View"

2. Step:
After some switching to the `Tilemap-Editor` and back to e.g. the `Scene-Editor`, the tile palette was still visible. (I stetted up multiple camera tabs, so I don't need to switch the CamView-States with the Dropdown-Box)
![](http://i.imgur.com/iqLUKN3.png)

3. Step:
I dragged & dropped the tile palette into the "Project View" (3)
So it looked like this:
![](http://i.imgur.com/gAHD4Bz.png)

4. Step
I did the same like in the second step -> tile palette disappeared 

I hope this could help you to find the bug. It seams to be that the bug appears randomly..
